datalad/distribution/install.py

```diff
@@ -285,8 +282,8 @@ class Install(Interface):
     @staticmethod
     @datasetmethod(name='install')
     def __call__(
+            source,
             path=None,
-            source=None,
```

Comment:
LOL!  but it is not the name (dataLAD) which matters but the command -- "apt-get INSTALL"